tutorials/pub-sub/README.md

```diff
@@ -331,7 +331,55 @@ dapr stop --app-id react-form
 
 <!-- END_STEP -->
 
-4. If you want to deploy this same application to Kubernetes, move onto the next step. Otherwise, skip ahead to the [How it Works](#How-it-Works) section to understand the code!
+4. If you want more insights into the Cloud Events being published on Redis streams, keep going.  If you want to deploy this same application to Kubernetes,  jump to the [Run in Kubernetes](#Run-in-Kubernetes). Otherwise, skip ahead to the [How it Works](#How-it-Works) section to understand the code!
+
+## Insights into the Cloud Events being published on Redis streams
+If you want to see the [Cloud Events](https://github.com/cloudevents/spec) that are being published on [Redis stream](https://redis.io/docs/data-types/streams/), follow the below steps: 
+
+1. Redis is running inside of a docker container, so to access it via CLI, pass the following command to a new terminal:
+
+<!-- STEP
+expected_stdout_lines:
+ - 127.0.0.1:6379>
+expected_stderr_lines:
+name: Redis Docker Image
+-->
```

Comment:
```suggestion
```
Let's omit this for now because we're having reliability issues, and I worry these IP address based outputs might vary too, which may lead to failures.  

The spirit of this is to be a doc update.